atomic_reactor/util.py

```diff
@@ -1037,9 +1037,17 @@ def are_plugins_in_order(plugins_conf, *plugins_names):
     return True
 
 
-def read_yaml(yaml_file_path, schema):
-    with open(yaml_file_path) as f:
-        data = yaml.safe_load(f)
+def read_yaml(yaml_source, schema, from_file=True):
```

Comment:
I think I prefer having the `open()` done by the caller, so that `yaml_source` is always an iterable we can pass to `yaml.safe_load()`.

Reply:
this wasn't changed, just so it is compatible, but I can make what you suggesting